### PR TITLE
Master port to OM - Output/Dashboard - part2

### PR DESCRIPTION
**Pull request by s7design**

### CODE REVIEW

File: scripts/test/Selenium/Output/Dashboard/EventsTicketCalendar.t [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+            Valid => 1,
+            Key   => 'Ticket::Frontend::AgentTicketPhone###DynamicField',
+            Value => \%SysConfigDynamicField,
+        );
+
+        # get customer user object
+        my $CustomerUserObject = $Kernel::OM->Get('Kernel::System::CustomerUser');
+
+        # create customer user
+        my $TestCustomerUser   = "SeleniumCustomer" . rand int(1000);
```
Comment (mgruner) [inline scripts/test/Selenium/Output/Dashboard/EventsTicketCalendar.t]: Please check in the Helper.pm, there is a method that gives you a test customer user and also cleans it up at the end, just like TestUserCreate().

File: scripts/test/Selenium/Output/Dashboard/EventsTicketCalendar.t [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            $TestCustomerUserID,
+            "Customer user $TestCustomerUser - created"
+        );
+
+        # create test ticket that will be shown on EventsTicketCalendar
+        my $ScriptAlias = $Kernel::OM->Get('Kernel::Config')->Get('ScriptAlias');
+        $Selenium->get("${ScriptAlias}index.pl?Action=AgentTicketPhone");
+
+        my $AutoCompleteString
+            = "\"$TestCustomerUser $TestCustomerUser\" <$TestCustomerUser\@localhost.com> ($TestCustomerUser)";
```
Comment (mgruner) [inline scripts/test/Selenium/Output/Dashboard/EventsTicketCalendar.t]: Please create the ticket directly in the DB. I'm having trouble with this, it cannot find the TicketID if I run selenium.

Comment (s7design) [inline scripts/test/Selenium/Output/Dashboard/EventsTicketCalendar.t]: Yes we will try to do on this way, to be honest we thought about it, but we had a problem with add Dynamic_filed to ticket in beckend, but I hope we will find how to do that.
all of that will be done as soon as possible.

**Comment by s7design** [comment]
Hi Martin,

I changed test as you suggested. I hope that they are much better now.

I also enabled appropriate  SysConfig item for each of test. Some of them is enabled by default, but maybe it is better to be sure that test will be successfully run. I had some situation on my local system that some of these SysConfig item were disabled and test was failed.

Regars
Zoran

**Comment by mgruner** [comment]
Merged, thank you! I just removed one duplicated my $LayoutObject in UserOnline.pm.

**Pull request closed without merging**